Removing Unity

HunterBarclay opened the pull request:
### Description
Completely deleting the Unity project, as well as supporting files and components

### Removed
- [x] Removed `/engine/`
- [x] Removed `/api/`
- [x] Removed `/scripts/`
- [x] Removed engine workflows
- [x] Removed api workflows

### Updated
- [x] Updated workflow names
- [x] Updated `README.md` (Temporary)
- [x] Updated Logo storage location

HunterBarclay committed:
Removing the everything

HunterBarclay committed:
Updated documentation

HunterBarclay committed:
Adjusting dark/light theme image

HunterBarclay committed:
Bit hacky, but hopefully works

HunterBarclay committed:
Not working, sticking with the original

BrandonPacewic commented on README.md:
```diff
@@ -1,111 +1,72 @@
 
 If you are a FIRST robotics student who just wants to use Synthesis, you *don't* need this repo. Simply **install the latest release of Synthesis from [synthesis.autodesk.com/download](https://synthesis.autodesk.com/download.html)**.
 
+[!IMPORTANT]
+Moving to [synthesis.autodesk.com]
```
```suggestion
> [!IMPORTANT]
> Moving to [synthesis.autodesk.com]
```

The important tag needs to be formatted in this way to appear properly.

BrandonPacewic commented on .github/workflows/BlackFormat.yml:
```diff
@@ -1,4 +1,4 @@
-name: Format Validation (Black)
+name: Fusion - Format Validation
```
```suggestion
name: Fusion - Black Format Validation
```

Tiny nitpick, should be consistent with other formatting workflow.

https://github.com/Autodesk/synthesis/blob/00bb1addcffae3eb14e7c00ae895d7cf48943232/.github/workflows/FissionESLintFormat.yml#L1

HunterBarclay replied:
Right, I thought we didn't called it ES Lint.

BrandonPacewic reviewed (changes requested):
I believe the `init.bat` and `init.sh` scripts at root need to be updated as well.

HunterBarclay committed:
Update .github/workflows/BlackFormat.yml

Co-authored-by: Brandon Pacewic <pacewicbrandon@gmail.com>

HunterBarclay committed:
Little readme tweak

HunterBarclay committed:
Update README.md

Co-authored-by: Brandon Pacewic <pacewicbrandon@gmail.com>

HunterBarclay committed:
Merge branch 'dev' into barclah/nuke

HunterBarclay committed:
Adjusted init scripts for now. We'll re-evaluate them once we determine what we are doing with the protocols directory.

HunterBarclay commented:
@BrandonPacewic Made the changes plus changing the header for the "other components" section of the README. I'm leaving in the init scripts for now. Once we decide if we're going to use the protocols folder, we'll remove/keep them. For now, I just removed all the setup for things we removed.

a-crowell reviewed (approved):
Excited to see this cleaned out. Almost 600,000 lines, wow.
